pyrometheus/__init__.py

```diff
@@ -178,8 +179,93 @@ def equilibrium_constants_expr(sol: ct.Solution, react: ct.Reaction, gibbs_rt):
 
 # {{{ Rate coefficients
 
-def arrhenius_expr(rate_coeff: ct.Arrhenius):
-    raise NotImplementedError()
+def rate_coefficient_expr(rate_coeff: ct.Arrhenius, t):
+    """This function returns rate-coefficient expressions"""
+    # Rate parameters
+    a = rate_coeff.pre_exponential_factor
+    b = rate_coeff.temperature_exponent
+    t_a = rate_coeff.activation_energy/ct.gas_constant
+    if b == 0.0 and t_a == 0.0:
+        # Constant rate
+        return a
+    elif b != 0.0 and t_a == 0.0:
+        # Weakly temperature-dependent rate
+        return a * t**b
+    elif b == 0.0 and a != 0.0 and t_a != 0.0:
+        # Classic Arrhenius rate
+        return p.Variable("exp")(np.log(a)-t_a/t)
+    else:
+        # Modified Arrhenius
+        return p.Variable("exp")(np.log(a)+b*p.Variable("log")(t)-t_a/t)
+
+
+def third_body_efficiencies_expr(sol: ct.Solution, react: ct.Reaction, c):
+    """This function returns an expression for the third-body rate factor"""
+    efficiencies = [react.efficiencies[sp] for sp in react.efficiencies]
+    indices_nondef = [sol.species_index(sp) for sp in react.efficiencies]
+    indices_default = np.delete(np.linspace(0, sol.n_species-1, sol.n_species,
+                                            dtype=int), indices_nondef)
```

Comment:
```suggestion
    indices_default = [i for i in range(sol.n_species if i not in indices_nondef]
```
(easier to read maybe? but: higher complexity)